### PR TITLE
Feature: Add support to CNAB 150

### PR DESCRIPTION
**Pull request by pablotdv**

### CODE REVIEW

File: gocnab.go [hunk 1]
```diff
@@ -65,6 +66,10 @@ func WithFinalControlCharacter(enabled bool) MarshalOptionFunc {
 	})
 }
 
+func Marshal150(vs ...interface{}) ([]byte, error) {
```
Comment (rafaeljusto) [inline gocnab.go:69]: Could you please add some docs to this exported function?

**Review by rafaeljusto — COMMENTED** [review]
Hey @pablotdv , thanks for the contribution! Only a small suggestion below.

Do you have a link for a Febraban doc regarding the CNAB 150 out of curiosity?

**Comment by pablotdv** [comment]
Hey @rafaeljusto 

Here is the documentation link https://www.banrisul.com.br/bob/data/Banrisul_LayoutDebito_versao05_22122008.pdf

**Comment by pablotdv** [comment]
Can you close the PR?

**Review by rafaeljusto — APPROVED** [review]

**Pull request merged**